### PR TITLE
#3271 - Ministry: Student Requests - Appeals

### PR DESCRIPTION
**Pull request by bidyashish**
Acceptance Criteria:

- [x] On left hand menu bar revise: "Appeals" to "Change requests"
- [X] Revise: "Appeals" to "Change requests"
- [x] Revise: "Requested appeals" to "Pending change requests"
- [x] Revise: "Make a determination......" to "Change requests that require ministry review."
- [x] Expand list options to the following and align the order with the mockup:
- [x] Revise: "Name" to "Given names" and remove the sorting function on given name
- [x] Add: "Last name" and add sorting option to the Last name field

image

### CODE REVIEW

File: sources/packages/web/src/views/aest/student/StudentApplicationAppeals.vue [hunk 1]
```diff
@@ -45,7 +45,13 @@
               </span>
             </template>
           </Column>
-          <Column field="fullName" header="Name" :sortable="true"> </Column>
+          <Column field="firstName" header="Given names">
+            <template #body="slotProps">
+              <span>{{ slotProps.data.firstName || "" }}</span>
```
Comment (dheepak-aot) [inline sources/packages/web/src/views/aest/student/StudentApplicationAppeals.vue:50]: We have formatter called `emptyStringFiller` please use that. 
image
image

**Comment by dheepak-aot** [comment]
Nice work. Just one comment. Please take a look at the failing E2E tests.

**Review by andrepestana-aot — APPROVED** [review]
LGTM

**Review by dheepak-aot — APPROVED** [review]
Thanks for making the changes. Looks good 👍

**Pull request merged**